week08_pomdp/practice_pytorch.ipynb

```diff
@@ -278,7 +294,7 @@
     "            if done:\n",
     "                break\n",
     "\n",
-    "        game_rewards.append(total_reward)\n",
+    "        game_rewards.append(total_reward / reward_scale)\n",
```

Comment:
This might cause bugs because it's a global variable that is captured in this function at the moment when this function is defined. If it is actually required, better make it a parameter.